packages/zoe/src/zoeService/installationStorage.js

```diff
@@ -123,6 +139,8 @@ export const makeInstallationStorage = (
         );
         return installation;
       },
+      // XXX is there a better way to declare that the final else throws?
+      // eslint-disable-next-line consistent-return
```

Comment:
Use `throw Fail` rather than `Fail` when needed to help ts

Reply:
thanks.

Reply:
Looks done. Resolving.